src/calculateRentalCost.js

```diff
@@ -4,7 +4,18 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const bigReduction = 50;
+  const smallReduction = 20;
+  const price = 40;
```

Comment:
Consider replacing the hardcoded numbers `50`, `20`, and `40` with constants that have descriptive names. This will improve the readability and maintainability of your code.